scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -21,7 +21,7 @@
             {% elif variant.category == "str" %}
               <th rowspan="2" colspan="1">Expansion support</th>
             {% else %}
-	      {% if variant.chromosome in ["MT","M"] %}
+	      {% if variant.category == "snv" and (variant.chromosome in ["MT","M"] or ("panel" in case.analysis_types or "wes" in case.analysis_types)) %}
```

Comment:
For improved readability, consider something like this: 

```
{% set is_mito = variant.chromosome in ["MT", "M"] %}
{% set is_panel_or_wes = "panel" in case.analysis_types or "wes" in case.analysis_types %}
```

And then twice this: 

`{% if variant.category == "snv" and (is_mito or is_panel_or_wes) %}`

Reply:
Sure! We actually have an `is_mitochondrial` populated from the controller already - much better to use it than have a separate `chrom` check.